src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/formViewNetworkWideValidationSchema.tsx

```diff
@@ -63,8 +64,11 @@ const getMachineNetworkValidationSchema = (protocolVersion: ProtocolVersion) =>
       .transform(transformNumber) as Yup.NumberSchema, //add casting to not get typescript error caused by nullable
   });
 
-const getIPValidationSchema = (protocolVersion: ProtocolVersion) => {
-  return getIpAddressValidationSchema(protocolVersion)
+const getIPValidationSchema = (protocolVersion: ProtocolVersion, allowsMultiple = false) => {
+  const baseValidation = allowsMultiple
+    ? getMultipleIpAddressValidationSchema
+    : getIpAddressValidationSchema;
+  return baseValidation(protocolVersion)
```

Comment:
IIUC, in case of multiple isNotLocalHostIpAddress and isNotCatchAllIPAddress validations will not work?
But anyhow I think the is catch all/is local host logic should move to the backend

Reply:
Added to validation also for fields that may be multiple (if they are not, is part of a separate validation)

https://github.com/openshift-assisted/assisted-ui-lib/pull/1658/commits/06f25b8c306f17b667aeff410caca95ffa40881f